### ISSUE TITLE
[Feature] Remove container

### ISSUE THREAD

**Issue by wyqydsyq** [gpouilloux/gnome-shell-extension-docker#21]
Sometimes my list of docker containers gets long, especially when doing stuff that creates ephemeral (temporary w/ autogenerated names) containers.

Currently I need to either `docker rm` the containers explicitly or `docker system prune` to get rid of the dead ones.

It would be nice to have an additional option for each container, "Remove" which stops and then removes the container. This would allow users to clean up their container list from the UI of this extension.

If this is done, would be nice to also have a "Remove stopped" button that does this on all stopped containers as well.

**Comment by gpouilloux** [comment]
There's already an option to remove a container if it is stopped.

**Comment by wyqydsyq** [comment]
Indeed, but when you have 20+ containers, manually removing each one can take a while, especially since the UI of this extension appears to be [blocked while an operation is in progress](https://github.com/gpouilloux/gnome-shell-extension-docker/issues/21).
